VKApp/VKApp/Extension/UIViewControllerExtension.swift

```diff
@@ -0,0 +1,27 @@
+// UIViewControllerExtension.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Extension ErrorAlert
+extension UIViewController {
+    // MARK: - Private Constants
+
+    private enum Constants {
```

Comment:
```suggestion
    private enum Constants {
```
эти константы не универсальные верни их в контроллер обратно